src/bci_build/package/golang/README.md.j2

```diff
@@ -4,19 +4,36 @@
 
 ## Description
 
-[Go](https://go.dev/) (a.k.a., Golang) is a statically-typed programming language, with syntax loosely derived from C. Go offers additional features such as garbage collection, type safety, certain dynamic-typing capabilities, additional built-in types (for example, variable-length arrays and key-value maps) as well as a large standard library.
+[Go](https://go.dev/) (a.k.a., Golang) is a statically-typed programming
+language, with syntax loosely derived from C. Go offers additional features
+such as garbage collection, type safety, certain dynamic-typing capabilities,
+additional built-in types (for example, variable-length arrays and key-value
+maps) as well as a large standard library.
+
+{%- if image.version.endswith('-openssl') %}
+
+## FIPS 140-2
+
+The image includes a FIPS 140-2 enabled Go wrapper that prefers using OpenSSL
+for cryptographic operations. This allows the use of FIPS 140-2 validated
```

Comment:
What about FIPS 140-3? It is mentioned that you can also use it here: https://www.suse.com/c/go-and-fips-140-2-140-3-certified-cryptography/

Reply:
I only moved this text from the end of README to the beginning. and according to https://documentation.suse.com/sles/15-SP4/html/SLES-all/cha-security-fips.html we do not have FIPS-140-3 yet.

Reply:
ah, it was FIPS-140 before. ok, I'll make it 140-3 for now.